packages/core/src/utils.ts

```diff
@@ -19,7 +19,7 @@ export function isError (obj: any): obj is ErrorUnion {
   return obj && ('_' in obj) && obj._ === 'error'
 }
 
-export function toObject<T extends BaseTdObject> ({ response }: ApiResponse<any, T>): T {
+export function toObject<T extends BaseTdObject> ({ response }: ApiResponse<any, T>): ApiResponse<any, T>['response'] {
```

Comment:
No, it's not correct.

Lets look at the following code:

```ts
const user = toObject(await airgram.api.getMe())
```

Result is supposed to be an `User` object. If we apply your changes it will be breaking change, because function will return `User | Error`.

If you want to receive an `Error` object in the response you can do it like this:

```ts
const user = (await airgram.api.getMe()).response
// or
const { response } = await airgram.api.getMe()
```